mantidimaging/core/io/loader/loader.py

```diff
@@ -144,6 +152,8 @@ def load(filename_group: FilenameGroup,
             angle_order = np.argsort(angles)
             angles = angles[angle_order]
             file_names = [file_names[i] for i in angle_order]
+    if shutter_count_file is not None:
+        load_shutter_counts(shutter_count_file)
```

Comment:
Is this needed, it looks like it happens again at line 166

Reply:
Thanks for noticing this, i think it'd left over from a rebase. Will remove

Reply:
I still see it there as a comment.

Reply:
will remove, sorry about that 😅